spec/controllers/course/lesson_plan/items_controller_spec.rb

```diff
@@ -2,11 +2,13 @@
 require 'rails_helper'
 
 RSpec.describe Course::LessonPlan::ItemsController, type: :controller do
-  let!(:instance) { Instance.default }
+  let(:instance_video) { nil }
+  let!(:instance) { create(:instance, *instance_video) }
```

Comment:
Minor nitpick, but usually this will be `*instance_traits` . Same for the course one below